portfolio/index.yaml

```diff
@@ -53,5 +53,173 @@ indexes:
   ancestor: no
   properties:
   - name: rootid
+  - name: score
+    direction: asc
+
+- kind: Comment
+  ancestor: no
+  properties:
+  - name: name
+  - name: time
+    direction: desc
+
+- kind: Comment
+  ancestor: no
+  properties:
+  - name: name
+  - name: name
```

Comment:
I don't understand this. Why is this needed?

Reply:
It's because for more complex queries, you need to create an index file with all the possible configurations of queries in advance. Since I'm filtering by 3 parameters and sorting by 4 in 2 orders there's a lot of entries :(. I was wondering whether I should add this feature because of this.